recipe/meta.yaml

```diff
@@ -63,6 +63,12 @@ test:
     - kwant.solvers
     - kwant.solvers.tests
     - kwant.tests
+  commands:
+    - pip check
+    - python -c "import kwant; kwant.test()"
```

Comment:
IIUC this file will automatically be run https://github.com/conda-forge/kwant-feedstock/blob/main/recipe/run_test.py

Reply:
Yeah, I'm aware that the tests are already run. I am mostly messing around to figure out how to run tests on osx-arm64